v2/tools/generator/internal/armconversion/arm_spec_interface.go

```diff
@@ -30,8 +30,8 @@ func checkPropertyPresence(o *astmodel.ObjectType, name astmodel.PropertyName) e
 func NewARMSpecInterfaceImpl(
 	idFactory astmodel.IdentifierFactory,
 	resource *astmodel.ResourceType,
-	spec *astmodel.ObjectType) (*astmodel.InterfaceImplementation, error) {
-
+	spec *astmodel.ObjectType,
+) (*astmodel.InterfaceImplementation, error) {
```

Comment:
I still dislike this style.

Reads line. Sees `,`. Aha! There's more on the next line. `)`. Nope! Fooled you.

Reply:
For whatever it's worth I agree with @theunrepentantgeek.

Reply:
I think the default formatter I'm getting is slightly more aggressive than `go fmt`…